ivy/functional/backends/jax/__init__.py

```diff
@@ -233,3 +233,12 @@ def closest_valid_dtype(type=None, /, as_native=False):
 from .experimental import *
 from . import control_flow_ops
 from .control_flow_ops import *
+
+
+# sub-backends
+try:
```

Comment:
Following the discussion on the previous PR:
"mainly because I don't want people adding new backends to be forced to add the sub_backends folder and it's __init__ file, whenever a backend supports a sub_backend they can start adding folders/files/code needed" 

I think keeping a consistent structure is better so it's clear for everyone, also aligns with the current design of Ivy API structure (although it's required to implement all functions in the backends), but on a higher level, keeping the same structure is better. 

Another reason, one of the internal imports may fail, and that way of checking if a package exists or not may actually fail and never give a proper Exception if the sub_backend failed to import one of the imports internally.